CHANGELOG.rst

```diff
@@ -16,6 +16,11 @@ Unreleased
 
 *
 
+[0.3.5] - 2020-07-21
```

Comment:
Nit: This is really 0.4.0.  It isn't a bug fix, but an addition.

Reply:
:ok_hand: